test/python/transpiler/test_pass_scheduler.py

```diff
@@ -408,40 +412,84 @@ def test_fresh_initial_state(self):
         self.passmanager.append(PassM_AP_NR_NP(argument1=1))
         self.passmanager.append(PassA_TP_NR_NP())
         self.passmanager.append(PassM_AP_NR_NP(argument1=1))
-        self.assertScheduler(self.circuit, self.passmanager,
+        self.assertScheduler(self.passmanager,
                              ['run analysis pass PassM_AP_NR_NP',
                               'self.argument1 = 2',
                               'run transformation pass PassA_TP_NR_NP',
                               'run analysis pass PassM_AP_NR_NP',
                               'self.argument1 = 2'])
 
+    def test_rollback_if_const_true(self):
+        """ Dump passes with a rollback_if true"""
+        self.passmanager.append(PassE_AP_NR_NP(3))
+        self.passmanager.append(PassE_AP_NR_NP(2), rollback_if=lambda property_set: True)
```

Comment:
a little confused by this and the following test. Isn't property_set supposed to be a dict? Why is it compared to True/False then?

Reply:
The `rollback_if` will return constant `True`.